lms/resources/_js_config/file_picker_config.py

```diff
@@ -11,17 +11,11 @@ class FilePickerConfig:
     def blackboard_config(cls, request, application_instance):
         """Get Blackboard files config."""
         files_enabled = application_instance.settings.get("blackboard", "files_enabled")
-        groups_enabled = application_instance.settings.get(
-            "blackboard", "groups_enabled"
-        )
 
         auth_url = request.route_url(Blackboard.route.oauth2_authorize)
         course_id = request.lti_params.get("context_id")
 
-        config = {
-            "enabled": files_enabled,
-            "groupsEnabled": groups_enabled,
-        }
+        config = {"enabled": files_enabled}
```

Comment:
`enabled` inside `settings.canvas` means "filesEnabled". Will get to that at some point.